blockly/src/utils/BlocklyCodeRunner.java

```diff
@@ -35,6 +37,24 @@ public class BlocklyCodeRunner {
   private static final Logger LOGGER = Logger.getLogger(BlocklyCodeRunner.class.getSimpleName());
   private static BlocklyCodeRunner instance;
 
+  /** List of whitelisted Blockly command method names without the trailing (); or parameters. */
+  private static final List<String> WHITELIST =
+      List.of(
+          "hero.move",
+          "hero.rotate",
+          "hero.interact",
+          "hero.pull",
+          "hero.push",
+          "hero.pickup",
+          "hero.shootFireball",
+          "hero.active",
+          "hero.isNearTile",
+          "hero.isNearComponent",
+          "hero.dropItem",
+          "hero.rest",
+          "hero.checkBossViewDirection",
+          "hero.moveToExit");
+
```

Comment:
gibt noch `loadLevel` und `loadNextLevel` (ohne `hero.`) 
Die stehen nicht im command sheet, da debug.
Sind in der Workshopanleitung festgehalten https://github.com/Dungeon-CampusMinden/Dungeon/blob/master/doc/produs_unterlagen/materials/java_workshop.md